src/mito2/src/worker/handle_manifest.rs

```diff
@@ -165,6 +169,26 @@ impl<S> RegionWorkerLoop<S> {
                 self.handle_region_edit(request).await;
             }
         }
+
+        if need_compaction {
```

Comment:
I think it should be a request option, it's not a common practice to schedule compaction every time when editing regions.

Reply:
After discussed with @evenyag , we decide to add a config value to determine the time interval of two successive compactions. To align with the old behavior, the default value of this config is 0: no restrictions on how compactions are scheduled. In this editing region scene, people can adjust this config accordingly, to not trigger too frequent compactions.